docs/snippets/react/list-story-with-unchecked-children.ts.mdx

```diff
@@ -1,10 +1,14 @@
+```ts
 // List.stories.tsx
 
 import React from 'react';
 import { List } from './List';
 import { Unchecked } from './ListItem.stories';
 
+const Template = (args) => <List {...args} />;
```

Comment:
@DaleSeo I think this should be:

```tsx
import { Story } from '@storybook/react';
import { List, ListProps } from './List'

const Template: Story<ListProps> = (args) => <List {...args} />;
...
```

WDYT?

Reply:
@shilman Thank you for pointing that out. I fixed it.